### DISCUSSION TITLE
Welcome to the team, Winona

### DISCUSSION THREAD

**Discussion by joewiz** [eXist-db/exist#4878] [Team Posts]
As nominated by @duncdrum and seconded by me, Winona Salesky (@wsalesky) has joined the team of xquery-contributors. 

Winona is an extremely talented digital library consultant and is an active developer of XQuery-based applications with eXist. See her [GitHub profile](@wsalesky), her [CV](http://wsalesky.com/cv/), her [gists](https://gist.github.com/wsalesky), and her [tweets](https://twitter.com/wsalesky). She also was an instructor at the NEH-funded [XQuery Summer Institute](http://xqueryinstitute.org/) at Vanderbilt University in 2014 and presented on one of her Vanderbilt projects [at Balisage](https://www.balisage.net/Proceedings/vol16/html/Gibson01/BalisageVol16-Gibson01.html) in 2015.

On behalf of the team, welcome, Winona!

**Comment by wsalesky** [comment]
Thanks @joewiz! I'm excited to be one of the team. I'm looking forward to collaborating with you all.
